bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/inbox/InboxFilterCriteria.java

```diff
@@ -26,61 +27,137 @@
     private final String bindingId;
     private final ThingTypeUID thingTypeUID;
     private final ThingUID thingUID;
+    private final String representationValue;
     private final DiscoveryResultFlag flag;
 
+    private InboxFilterCriteria(String bindingId, ThingTypeUID thingTypeUID, ThingUID thingUID,
```

Comment:
It seems there are several constructors in that class. Couldn't we change all the constructors that them call one implementation (e.g. `this(bindingId, thingTypeUID, thingUID, null, null)`) instead of initialize members in every constructor implementation itself?